packages/@coorpacademy-components/src/atom/select/index.js

```diff
@@ -42,7 +42,7 @@ const Select = (props, legacyContext) => {
     modified = false,
     error = false,
     title: propTitle,
-    moreAriaLabel
+    'aria-label': ariaLabel
```

Comment:
changed prop's name because the icon can have many trads now (more, dropdown answer, possibly others later).